u2f-ref-code/java/src/com/google/u2f/server/impl/attestation/android/AuthorizationList.java

```diff
@@ -23,6 +23,10 @@ public Algorithm getAlgorithm() {
     return algorithm;
   }
 
+  public boolean isEmpty() {
```

Comment:
Seems funny to add dead code.. something to keep an eye on, I won't fuss about it here.